schemas/iso19110/src/main/plugin/iso19110/loc/eng/codelists.xml

```diff
@@ -72,7 +72,7 @@
     <entry>
       <code>processor</code>
       <label>Processor</label>
-      <description>Party wha has processed the data in a manner such that the resource has been
+      <description>Party what has processed the data in a manner such that the resource has been
```

Comment:
This does not seem to be correct?  Should it be "Party that has..."

Reply:
@ianwallen indeed the other entries descriptions use `that` instead of `what`